### PR TITLE
Update deps

### LINKED ISSUE #168: Migrate to `@stylelint/remark-preset`
We can use our sharable configuration for remark-lint: [`@stylelint/remark-preset`](https://github.com/stylelint/remark-preset).

https://github.com/stylelint/stylelint-config-standard/blob/809dd01497099c1ee6d2098988781d8dbfe4ebae/package.json#L57-L58

For example, stylelint/stylelint use the configuration:
https://github.com/stylelint/stylelint/blob/ba3aa5022d1813a85516bcfe26a27936aad66b6c/package.json#L76

### LINKED ISSUE #169: Missing LICENSE file
> Clearly describe the documentation issue

The license file is missing, although the README refers to the file.

https://github.com/stylelint/stylelint-config-standard/blob/809dd01497099c1ee6d2098988781d8dbfe4ebae/README.md#license

> What solution would you like to see?

Perhaps, our license is **MIT**, so I think we can prepare it easily.

https://github.com/stylelint/stylelint-config-standard/blob/809dd01497099c1ee6d2098988781d8dbfe4ebae/package.json#L11

### PR DESCRIPTION
**Pull request by jeddy3**
> Which issue, if any, is this issue related to?

Closes #169
Closes #168

> Is there anything in the PR that needs further explanation?

I figured this was a good time to bring the package up to date as we'll likely be putting out a new release alongside the next stylelint release.

Using the stylelint remark config caught the missing license, so I've addressed by issues in this pull request.

### CODE REVIEW

**Review by ybiquitous — COMMENTED** [review]
@jeddy3 Thanks!

It seems `is-ci` and `pinst` are missing in `devDependencies`. No problems?

**Review by ybiquitous — APPROVED** [review]
Looks nice! 👍

File: package.json [hunk 1]
```diff
@@ -7,64 +7,69 @@
...
+    "postpublish": "pinst --enable",
     "release": "np",
     "test": "jest",
     "watch": "jest --watch"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
```
Comment (ybiquitous) [inline package.json:37]: Oh, sorry. Can we remove this old `husky` config?

```suggestion
```

**Comment by jeddy3** [comment]
> Oh, sorry. Can we remove this old husky config?

Oops. We'll need to remove this from the main stylelint repo too as it's still in there too.

**Review by ybiquitous — APPROVED** [review]
Thank you 👍

**Pull request merged**